packages/ui-kit/package.json

```diff
@@ -32,14 +32,19 @@
     "lodash.escape": "4.0.1",
     "lodash.throttle": "4.1.1",
     "prism-react-renderer": "1.2.1",
-    "prop-types": "15.8.1",
```

Comment:
In theory `prop-types` should still be a dependency, so that projects that are not using TS can still validate the props.

For that, we need to use the `babel-plugin-typescript-to-proptypes`.

https://github.com/commercetools/ui-kit/blob/482e977c7f0cf3dcd342731c965e9016da5dc417/babel.config.js#L15